src/baselines.jl

```diff
@@ -91,18 +94,31 @@ end
 
 function selection_cut(wf::RDWaveform, base_uplim::Real, base_lolim::Real)
     # baseline start cut
-    base_start = wf.signal[1]
-    peak_index = risepoint(wf)
+    base_start = wf.signal[1] # value of waveform at the beginning
+    peak_index = risepoint(wf) # value of wf right before signal rising i.e. end of baseline
     baseline = wf.signal[begin:peak_index]
+    # is this supposed to remove pileup? i think it is base_slope() due to it not being flat
```

Comment:
Yes, this seems like a pileup cut on the baseline (making sure that the first baseline value is within the interval `base_lolim` to `base_uplim` and that it has no large slope (such that the mean is somewhat close to the first value)

Reply:
Thanks! The "baseline catalog" part of LegendGeSim is underdeveloped for now because it doesn't work for HADES data, as it does not usually contain all-baseline "empty" waveforms, unlike L200 data where in some trigger modes when one detectors triggers all record waveforms. So I imagine we will come back to this later when moving to L200 data simulation. What happens now with the `NoiseFromData` mode, it simply extracts noise and offset from there and adds that to the waveform, which is basically really same as `NoiseFromSim` in that it only adds this high frequency Gaussian noise.